### PR TITLE
Feature/reference-solar-cell

### PR DESCRIPTION
**Pull request by KarlSchu**
Analog input from ADC1:A2 ist stored and printed as ReferenceCellData in CarState.

### CODE REVIEW

**Comment by andreaskuster** [comment]
Please merge #158 after you checked my comment, and re-base this PR then, otherwise I have to review everything twice. I suggest you branch off from one branch to the next sequentially, which allows to review the diff instead if you plan to work on multiple PRs in parallel.

File: CHANGELOG_HW.md [hunk 1]
```diff
@@ -4,25 +4,35 @@
...
 |                    |          |          |
 
 ### SteeringWheel Board
-|                    | Old      | New      |
+
+| Old                | New      |          |
 | ------------------ | -------- | -------- |
-| SD Card detect | % | IOExt2:5 |
-|ILI C/D Low active	| R to GND	| R to VCC |  
+| SD Card detect     | %        | IOExt2:5 |
```
Comment (andreaskuster) [inline CHANGELOG_HW.md:24]: What does `%` mean in this context?

File: CHANGELOG_HW.md [hunk 2]
```diff
@@ -4,25 +4,35 @@
...
+
+| Old     | New                      |                                             |
+| ------- | ------------------------ | ------------------------------------------- |
+| xbee    | GND, VCC, TX, RX on Plug | GND,VCC,Tx, RX, SET on Plug                 |
+| LTC4311 | Integrate into mainboard | remark: near ESP32 SDA,SCL or near MCP23017 |
+
 |                    | Old      | New      |
 | ------------------ | -------- | -------- |
-| LTC4311 | GND, VCC, TX, RX on Plug | GND,VCC,Tx, RX, SET on Plug |
+| SD Card detect | % | IOExt2:5 |
```
Comment (andreaskuster) [inline CHANGELOG_HW.md:37]: Same as above

**Review by andreaskuster — COMMENTED** [review]
LGTM

File: CHANGELOG_HW.md (hunk 2 above)
Comment (KarlSchu) [reply CHANGELOG_HW.md:37]: "nothing" old --> it is new

**Review by andreaskuster — APPROVED** [review]

File: CHANGELOG_HW.md (hunk 1 above)
Comment (andreaskuster) [reply CHANGELOG_HW.md:24]: I don't think that I get this, but we can discuss it tonight :)

**Pull request merged**